package.json

```diff
@@ -87,6 +87,13 @@
     "migrations:build": "pnpm --filter design-system-migrations build",
     "migrations:lint": "pnpm --filter design-system-migrations lint",
     "migrations:lint:fix": "pnpm --filter design-system-migrations lint:fix",
+    "eslint:play": "pnpm --filter design-system-eslint play",
+    "eslint:start": "pnpm --filter design-system-eslint start",
+    "eslint:build": "pnpm --filter design-system-eslint build",
+    "eslint:lint": "pnpm --filter design-system-eslint lint",
+    "eslint:lint:fix": "pnpm --filter design-system-eslint lint:fix",
+    "eslint:test": "pnpm --filter design-system-eslint test",
+    "eslint:test:watch": "pnpm --filter design-system-eslint test:watch",
```

Comment:
These scripts should be renamed to `eslint:unit` and `eslint:unit:watch` to align them with all the other packages.